docs/index.md

```diff
@@ -50,155 +55,91 @@ Compose has commands for managing the whole lifecycle of your application:
 ## Compose documentation
 
 - [Installing Compose](install.md)
+- [Getting Started](gettingstarted.md)
 - [Get started with Django](django.md)
 - [Get started with Rails](rails.md)
 - [Get started with WordPress](wordpress.md)
 - [Command line reference](./reference/index.md)
 - [Compose file reference](compose-file.md)
 
-## Quick start
-
-Let's get started with a walkthrough of getting a simple Python web app running
-on Compose. It assumes a little knowledge of Python, but the concepts
-demonstrated here should be understandable even if you're not familiar with
-Python.
-
-### Installation and set-up
-
-First, [install Docker and Compose](install.md).
-
-Next, you'll want to make a directory for the project:
-
-    $ mkdir composetest
-    $ cd composetest
-
-Inside this directory, create `app.py`, a simple Python web app that uses the Flask
-framework and increments a value in Redis. Don't worry if you don't have Redis installed, docker is going to take care of that for you when we [define services](#define-services):
-
-    from flask import Flask
-    from redis import Redis
+## Features
 
-    app = Flask(__name__)
-    redis = Redis(host='redis', port=6379)
+#### Preserve volume data
 
-    @app.route('/')
-    def hello():
-        redis.incr('hits')
-        return 'Hello World! I have been seen %s times.' % redis.get('hits')
+Compose preserves all volumes used by your services. When `docker-compose up`
+runs, if it finds any containers from previous runs, it copies the volumes from
+the old container to the new container. This process ensures that any data
+you've created in volumes isn't lost.
 
-    if __name__ == "__main__":
-        app.run(host="0.0.0.0", debug=True)
 
-Next, define the Python dependencies in a file called `requirements.txt`:
+#### Only recreate containers that have changed
 
-    flask
-    redis
+Compose caches the configuration used to create a container. When you
+restart a service that has not changed, Compose re-uses the existing
+containers. Re-using containers means that you can make changes to your
+environment very quickly.
 
-### Create a Docker image
 
-Now, create a Docker image containing all of your app's dependencies. You
-specify how to build the image using a file called
-[`Dockerfile`](http://docs.docker.com/reference/builder/):
+#### Variables and moving a composition to different environments
 
-    FROM python:2.7
-    ADD . /code
-    WORKDIR /code
-    RUN pip install -r requirements.txt
-    CMD python app.py
+> New in `docker-compose` 1.5
```

Comment:
Leave this out please.  Listing of new features happen in release notes and changelogs.  It is distracting to put this here and only informational for the set of users that previously used compose --- again those folks unlikely to use this intro.